src/dashboard/apps/core/models.py

```diff
@@ -34,6 +35,51 @@ class Entity(DashboardBase):
         symmetrical=False,
         related_name="proxies",
     )
+    company_type = models.CharField(
+        _("type"),
+        max_length=255,
+        help_text=_(
+            "entreprise/ collectivité locale, ECPI, Association, copropriété, ..."
```

Comment:
I've seen that elsewhere: is it intended?

```suggestion
            "entreprise, collectivité locale, ECPI, Association, copropriété, ..."
```

Reply:
No, that's a typo :-/ 
I think it's also present in the linked migration file